relays/bin-substrate/src/rialto_millau/westend_headers_to_millau.rs

```diff
@@ -61,13 +62,29 @@ pub async fn run(
 	westend_client: WestendClient,
 	millau_client: MillauClient,
 	millau_sign: MillauSigningParams,
-	metrics_params: Option<relay_utils::metrics::MetricsParams>,
+	metrics_params: relay_utils::metrics::MetricsParams,
 ) -> Result<(), String> {
 	crate::finality_pipeline::run(
 		WestendFinalityToMillau::new(millau_client.clone(), millau_sign),
 		westend_client,
-		millau_client,
-		metrics_params,
+		millau_client.clone(),
+		relay_utils::relay_metrics(
+			finality_relay::metrics_prefix::<WestendFinalityToMillau>(),
+			metrics_params.address,
+		)
+		.standalone_metric(FloatJsonValueMetric::new(
+			"https://api.coingecko.com/api/v3/simple/price?ids=Polkadot&vs_currencies=usd".into(),
```

Comment:
Is Millau not listed on CoinGecko yet? We should look into it 😏

On a more serious note, I would make a `NOTE` here indicating why we're using DOT/KSM price feeds here just like you noted in the PR description